test/controllers/AnyMoreChangesControllerSpec.scala

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
Can we maintain the same package structure of controllers/vary/ for the Spec file too. The same goes for view spec, formProvider and Id file please

Reply:
Fixed